Rakefile

```diff
@@ -1,22 +1,6 @@
-#!/usr/bin/env rake
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
 
-begin
-  require 'bundler/setup'
-rescue LoadError
-  puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
+RSpec::Core::RakeTask.new(:spec) do |task|
+  task.rspec_opts = ['--color', '--format', 'documentation']
 end
```

Comment:
Are the rest of the defaults really not necessary? (I realize some of them are Rails-only)